package.json

```diff
@@ -2,46 +2,46 @@
   "name": "drone-ui",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/drone/drone-ui.git"
   },
   "author": "",
-  "license": "ISC",
+  "license": "Apache",
```

Comment:
Thanks, that was just a quick fix I did without proper research. It's updated in this PR.